build/testing/cypress/integration/examples/actions.spec.js

```diff
@@ -0,0 +1,272 @@
+/// <reference types="Cypress" />
+
+context('Actions', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/commands/actions')
+  })
+
+  // https://on.cypress.io/interacting-with-elements
+
+  it('.type() - type into a DOM element', () => {
+    // https://on.cypress.io/type
+    cy.get('.action-email')
+      .type('fake@email.com').should('have.value', 'fake@email.com')
+
+      // .type() with special character sequences
+      .type('{leftarrow}{rightarrow}{uparrow}{downarrow}')
```

Comment:
What is the benefit of having all of these example files?

Leaving these files hanging around is confusing to me because they're aren't used (adds clutter) and they're only going to get stale as cypress updates. Just looking at this PR, it's not clear to me what is relevant to our code base.

I would prefer folks reference real documentation to learn how to write tests as opposed to old file hanging around our repository.

Reply:
I agree with you so I've removed unused examples.